templates/mon_users_and_limits.sql.j2

```diff
@@ -18,13 +18,13 @@ GRANT SELECT ON TABLE MonMetrics.DefinitionDimensions TO monasca_api;
 GRANT ALL ON TABLE MonAlarms.StateHistory TO monasca_api;
 
 {% for item in vertica_users %}
-CREATE USER {{ item.username }} IDENTIFIED BY '{{ item.password }}';
+CREATE USER '{{ item.username }}' IDENTIFIED BY '{{ item.password }}';
```

Comment:
I finally got around to testing this yesterday and Vertica SQL does not like any of the extra quotes you added in this file.